documentation/sdk/docs/how_tos/advanced_transactions/claim_transaction.mdx

```diff
@@ -0,0 +1,57 @@
+---
+title: Claim Transaction
+description: 'How to claim funds'
+image: /img/logo/iota_mark_light.png
+keywords:
+- how to
+- time-lock
+- storage deposit return
+- unlock conditions
+- claim output
+- nodejs
+- python
+- rust
+---
+
+import CodeBlock from '@theme/CodeBlock';
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+import NodejsCode from "!!raw-loader!../../../../../bindings/nodejs/examples/how_tos/advanced_transactions/claim_transaction.ts";
+import PythonCode from "!!raw-loader!../../../../../bindings/python/examples/how_tos/advanced_transactions/claim_transaction.py";
+import RustCode from "!!raw-loader!../../../../../sdk/examples/how_tos/advanced_transactions/claim_transaction.rs";
+
+You can use the SDK to send transaction with funds that have to be claimed (for example a [micro transaction](send_micro_transaction.mdx)). This How-To will show you how. It will claim all claimable ouputs in your account.
```

Comment:
This How-To will show you how. 
Very useful :carlos:

Reply:
It is correct tho. Some funds need to be claimed. This How-To will show you how (to do that :trollface: )

```suggestion
You can use the SDK to send transaction with funds that have to be claimed (for example a [micro transaction](send_micro_transaction.mdx)). This How-To will show you how to claim all claimable ouputs in your account.
```